great_expectations/expectations/metrics/util.py

```diff
@@ -373,7 +373,7 @@ def __getitem__(self, key: Any) -> Any:
 
 def get_sqlalchemy_column_metadata(  # noqa: C901 # FIXME CoP
     execution_engine: SqlAlchemyExecutionEngine,
-    table_selectable: sqlalchemy.Select,
+    table_selectable: sqlalchemy.Select | str,
```

Comment:
I don't think this is ever actually an `sqlalchemy.Select` but I can't guarantee that 🍝